src/app/vault/vault.component.ts

```diff
@@ -98,7 +98,9 @@ export class VaultComponent implements OnInit, OnDestroy {
                         await this.addFolder();
                         break;
                     case 'focusSearch':
-                        (document.querySelector('#search') as HTMLInputElement).focus();
+                        let search = document.querySelector('#search') as HTMLInputElement;
+                        search.focus();
+                        search.setSelectionRange(0, (search.value || '').length);
```

Comment:
Why not just `.select()`?

Reply:
Because we don't thought about it :(
Tested and works correctly. Do you need a new commit or do yourself?
thanks